openpype/hosts/maya/plugins/publish/validate_resolution.py

```diff
@@ -0,0 +1,54 @@
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin
+)
+from maya import cmds
+from openpype.hosts.maya.api.lib import reset_scene_resolution
+
+
+class ValidateResolution(pyblish.api.InstancePlugin,
```

Comment:
I am thinking if we can possibly have an abstract class for this validator as the code looks super similar to the Max Hosts. It can be used across different hosts eventually.